engine/src/w32-clipboard.cpp

```diff
@@ -300,23 +300,46 @@ MCStringRef MCWin32RawClipboardCommon::DecodeTransferredFileList(MCDataRef p_dat
 		MCStringCreateWithBytes(t_bytes, t_path_char_count*2, kMCStringEncodingUTF16, false, t_decoded);
```

Comment:
Make t_decoded an MCAutoStringRef now - then you won't have to release it manually.